examples/powertools-examples-core/README.md

```diff
@@ -0,0 +1,138 @@
+# CoreUtilities
```

Comment:
These and the reste of the concrete samples have simply been lifted from `main` from the linked samples repository.

https://github.com/aws-samples/aws-lambda-powertools-examples